### DISCUSSION TITLE
Implement Testcase for cli-apps using bubble-tea list-simple model

### DISCUSSION THREAD

**Discussion by Kameshvenkatesh** [charmbracelet/bubbletea#1062] [Q&A]
When implementing test-case for a cli-apps using testing go package, how to simulate the userinput which we get from list-simple bubbletea model, To brief the context I have developed a cli tool and integrated bubbletea list-simple model to get user input as a list of options (For example Yes or No) I need to simulate the user input which I get from the bubbeltea list options model, Any idea on how to implement testcase in this scenario ?

**Comment by Broderick-Westrope** [comment]
Hopefully this reply isn't too late, but you can use the teatest to test bubbletea models. [This](https://charm.sh/blog/teatest/) blog post has more info.
